packages/packages/documents/src/hooks/use-documents-actions.ts

```diff
@@ -0,0 +1,22 @@
+import { useCallback } from 'react';
+import { openRoute, runCommand } from '@elementor/v1-adapters';
+
+export function useDocumentsActions() {
+	const save = useCallback( () => {
```

Comment:
Try to use `dispatch`, and use the conventions of https://redux-toolkit.js.org/api/createAsyncThunk

Reply:
No